framework/src/media/utils/MediaUtils.cpp

```diff
@@ -323,8 +332,7 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 				meddbg("malloc failed error\n");
 				return false;
 			}
-			int ret;
-			ret = fseek(fp, -1, SEEK_CUR);
+			ret = fseek(fp, -2, SEEK_CUR);
```

Comment:
I think calling fseek before malloc is more comfortable.
If malloc is failed, it will skip header forever.

Reply:
currently, If either of them is fail, the result is the same.
I will cleanup code later.